src/components/Form/Form.js

```diff
@@ -0,0 +1,164 @@
+import './Form.css';
+import Button from "../button/Button";
+
+const Form = () => {
+    const [formState, setFormState] = useState({
+        firstname: '',
+        lastname: '',
+        age: '',
+        postalCode: '',
+        frequency: 'weekly',
+        period: '',
+        message: '',
+        conditions: false,
+    });
+
+    function handleChange(event) {
+        const changedFieldName = event.target.name;
+        const newValue = event.target.type === 'checkbox' ? event.target.checked : event.target.value;
+        setFormState({
+            ...formState,
+            [changedFieldName]: newValue,
+        });
+    }
+
+    function submit() {
+        console.log(formState)
+    }
+
+    return (
+        <form onSubmit={submit}>
+            <FormField
+                type='text'
+                id='firstname-field'
+                name='firstname'
+                text='voornaam'
+                value={formState.firstname}
+                changeHandler={handleChange}
+            />
+            <FormField
+                type='text'
+                id='lastname-field'
+                name='lastname'
+                text='achternaam'
+                value={formState.lastname}
+                changeHandler={handleChange}
+            />
+            <FormField
+                type='number'
+                id='age-field'
+                name='age'
+                text='leeftijd'
+                value={formState.age}
+                changeHandler={handleChange}
+                min='0'
+                max='150'
+            />
+            <FormField
+                type='text'
+                id='postal-code-field'
+                name='postalCode'
+                text='postcode'
+                value={formState.postalCode}
+                changeHandler={handleChange}
+            />
+            <label
+                htmlFor='delivery-frequency-select-box'
+            >
+                bezorgfrequentie
+            </label>
+
+            <select
+                name='frequency'
+                id='delivery-frequency-select-box'
+                value={formState.frequency}
+                onChange={handleChange}>
+                <option value='weekly'>iedere week</option>
+                <option value='two weekly'>om de week</option>
+                <option value='monthly'>iedere maand</option>
+            </select>
+
+            <div className='period-container'>
+                <div
+                    className='period-radio-buttons'
+                >
+                    <label
+                        htmlFor='day-radio'
+                    >
+                        overdag
+                    </label>
+                    <input
+                        type='radio'
+                        id='day-radio'
+                        name='period'
+                        value='day'
+                        checked={formState.period === 'day'}
```

Comment:
Als het goed is hoef je dit niet zelf te doen. Zolang je meerdere radiobuttons hebt met dezelfde naam, dan worden die checked waardes vanzelf goed gezet. Als je er ééntje default wil laten aanvinken dan kan dat zo:
```suggestion
                        defaultChecked={true}
```
of als shorthand
```suggestion
                        defaultChecked
```